src/components/POLine/RelatedInvoiceLines/constants.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-shadow */
```

Comment:
why we need it here ?

Reply:
![Screenshot 2023-09-20 at 2 26 45 PM](https://github.com/folio-org/ui-orders/assets/116072773/9e87373c-22aa-4ab4-8fd3-b9abfb1b6b18)
To resolve the error

Reply:
Hm, I think you can pass in r`esult_formatter` `item` instead of `invoice` and error will be solved.